yarn.lock

```diff
@@ -2,6 +2,71 @@
 # yarn lockfile v1
 
 
+"@babel/code-frame@7.0.0-beta.31":
```

Comment:
is it coming from jest?

Reply:
babel-jest I suppose

Reply:
Hmmm was thinking whether it would help us with https://github.com/babel/minify/pull/487#discussion_r138674114